app/views/contexts/list_collector.py

```diff
@@ -6,12 +6,13 @@
 
 
 def build_list_items_summary_context(
-    rendered_block, schema, answer_store, list_store, language
+    list_collector_block, schema, answer_store, list_store, language
 ):
-    list_name = rendered_block['for_list']
+    list_name = list_collector_block['for_list']
     list_item_ids = list_store[list_name].items
     list_answer_ids = [
-        answer['id'] for answer in rendered_block['add_block']['question']['answers']
+        answer['id']
+        for answer in list_collector_block['add_block']['question']['answers']
```

Comment:
This breaks if there are variants in the `add_block`. I've had a look and I don't think `list_answer_ids` is needed as it's only used to add an `answers` key to the list item, which is not used now that `item_title` exists.